packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -109,8 +132,34 @@ export const prepareStatusManager = zone => {
        * Add a new transaction with ADVANCED status
        * @param {CctpTxEvidence} evidence
        */
-      advance(evidence) {
-        recordPendingTx(evidence, PendingTxStatus.Advanced);
+      advancing(evidence) {
```

Comment:
I think `advance` was correct. That's the transition event that will change the state to Advancing.

Consider having a naming scheme for all transition events. "send" is common in state FSMs. So here could be `sendAdvance`

Reply:
restored `advance`

just added a TODO re naming scheme